deploy/objects/AppManagerWrapper.js

```diff
@@ -2,22 +2,22 @@ import decodeLogs from '../../helpers/decodeLogs';
 import encodeCall from 'zos-core/test/helpers/encodeCall';
 const UpgradeabilityProxyFactory = artifacts.require('UpgradeabilityProxyFactory');
 
-export default class ProjectControllerManager {
-  constructor(controller, owner) {
+export default class AppManagerWrapper {
```

Comment:
We may want to add wrapper functions for BaseAppManager's `upgradeTo` and `upgradeToAndCall` mainly and perhaps other functions as well.